torch/csrc/jit/codegen/cuda/executor.cpp

```diff
@@ -522,9 +536,20 @@ std::vector<at::Tensor> FusionExecutor::runFusion(
     executor_utils::validateVectorizedTensors(
         &fusion_, inputs, outputs, lowered_, expr_eval);
 
+    auto alias_indices = fusion_.getInputAliasIndices();
+
+    // TODO: Update this as well;
     if (outputs.empty() || outputs.size() != fusion_.outputs().size()) {
```

Comment:
So we just ignore the provided outputs here?

Reply:
We can also assert here when the provided output doesn't match.